arch/sim/Kconfig

```diff
@@ -98,7 +98,16 @@ config SIM_X8664_MICROSOFT
 endchoice
 
 config SIM_WALLTIME
-	bool "Execution simulation in near real-time"
+	bool "Execute the simulation in near real-time"
+	default n
+
+if SIM_WALLTIME
+choice
+	prompt "Execute the simulation in near real-time"
```

Comment:
Should we reword the prompt to avoid the dup with line 101?

Reply:
Yes, and we should also update the README where SIM_WALLTIME is mentioned.